.github/workflows/docker-publish.yml

```diff
@@ -0,0 +1,87 @@
+name: Release Docker image to GHCR
+
+on:
+  # TODO: wire into release process
```

Comment:
I think the [macbre/push-to-ghcr](https://github.com/marketplace/actions/push-to-ghcr) action will do this automatically - ideally if we're doing a numbered version release, we'd get a container tagged with that same version, otherwise push to the `latest` tag. 

That action doesn't build both x86_64 and arm64 containers as yours does, though.

Reply:
nevermind, that action doesn't expose digests, which are needed for an attestation.